// nocopyrightsounds-api/dist/esm/helpers/enums.js
export var Genre
;(function (Genre) {
    Genre[(Genre['Bass'] = 1)] = 'Bass'
    Genre[(Genre['Chill'] = 2)] = 'Chill'
    Genre[(Genre['DrumBass'] = 3)] = 'DrumBass'
    Genre[(Genre['Drumstep'] = 4)] = 'Drumstep'
    Genre[(Genre['Dubstep'] = 5)] = 'Dubstep'
    Genre[(Genre['EDM'] = 6)] = 'EDM'
    Genre[(Genre['Electronic'] = 7)] = 'Electronic'
    Genre[(Genre['FutureHouse'] = 8)] = 'FutureHouse'
    Genre[(Genre['Hardstyle'] = 9)] = 'Hardstyle'
    Genre[(Genre['House'] = 10)] = 'House'
    Genre[(Genre['Indie'] = 11)] = 'Indie'
    Genre[(Genre['MelodicDubstep'] = 12)] = 'MelodicDubstep'
    Genre[(Genre['Unknown'] = 13)] = 'Unknown'
    Genre[(Genre['Trap'] = 14)] = 'Trap'
    Genre[(Genre['GlitchHop'] = 15)] = 'GlitchHop'
    Genre[(Genre['Phonk'] = 16)] = 'Phonk'
    Genre[(Genre['FutureBass'] = 17)] = 'FutureBass'
    Genre[(Genre['BassHouse'] = 18)] = 'BassHouse'
})(Genre || (Genre = {}))
export var Mood
;(function (Mood) {
    Mood[(Mood['Angry'] = 1)] = 'Angry'
    Mood[(Mood['Dark'] = 2)] = 'Dark'
    Mood[(Mood['Dreamy'] = 3)] = 'Dreamy'
    Mood[(Mood['Epic'] = 4)] = 'Epic'
    Mood[(Mood['Euphoric'] = 5)] = 'Euphoric'
    Mood[(Mood['Energetic'] = 6)] = 'Energetic'
    Mood[(Mood['Fear'] = 7)] = 'Fear'
    Mood[(Mood['Funny'] = 8)] = 'Funny'
    Mood[(Mood['Glamorous'] = 9)] = 'Glamorous'
    Mood[(Mood['Gloomy'] = 10)] = 'Gloomy'
    Mood[(Mood['Happy'] = 11)] = 'Happy'
    Mood[(Mood['Hopeful'] = 12)] = 'Hopeful'
    Mood[(Mood['LaidBack'] = 13)] = 'LaidBack'
    Mood[(Mood['Mysterious'] = 14)] = 'Mysterious'
    Mood[(Mood['Peaceful'] = 15)] = 'Peaceful'
    Mood[(Mood['Quirky'] = 16)] = 'Quirky'
    Mood[(Mood['Relaxing'] = 17)] = 'Relaxing'
    Mood[(Mood['Restless'] = 18)] = 'Restless'
    Mood[(Mood['Romantic'] = 19)] = 'Romantic'
    Mood[(Mood['Sad'] = 20)] = 'Sad'
    Mood[(Mood['Scary'] = 21)] = 'Scary'
    Mood[(Mood['Sexy'] = 22)] = 'Sexy'
    Mood[(Mood['Suspense'] = 23)] = 'Suspense'
    Mood[(Mood['Weird'] = 24)] = 'Weird'
})(Mood || (Mood = {}))
